psi4/driver/procrouting/sapt/sapt_util.py

```diff
@@ -122,8 +122,7 @@ def print_sapt_hf_summary(data, name, short=False, delta_hf=False):
         return ret
 
 
-def print_sapt_dft_summary(data, name, short=False):
-
+def print_sapt_dft_summary(data, name, do_dft=True, short=False):
```

Comment:
```suggestion
def print_sapt_dft_summary(data, name, *, do_dft=True, short=False):
```
If there are any other changes to be made, I'd add a star here so that callers have to name their arguments and don't risk mixing up do_dft and short.